src/Translator.php

```diff
@@ -837,4 +846,21 @@ private function recursiveKeySort(array &$arr): void
             }
         }
     }
+
+    /**
+     * The tx binary doesn't appear to correctly read from ~/.transifexrc, even though it will correctly
+     * write to it after prompting for it if it's missing. This is a workaround to read the token and
+     * use it a TX_TOKEN environment variable, which the tx binary will use.
+     */
+    private function readTxToken(): void
+    {
+        if (!$this->exec('if [ -f ~/.transifexrc ]; then echo 1; else echo 0; fi')) {
```

Comment:
Please use PHP here rather than using bash.

Sorry I didn't notice this last time.

Reply:
Need to use bash as we're using `~` and we need it to resolve to the user home path

Reply:
You can use `$_SERVER['HOME']` or `getenv('HOME')` or add `symfony/filesystem` as a dependency and use `Path::canonicalize()`

Reply:
Updated